### PR TITLE
(feat) Listing, drilling and home page for business glossary

### PR DESCRIPTION
**Pull request by saxo-lalrishav**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-web-react/src/utils/test-utils/TestPageContainer.tsx [hunk 1]
```diff
@@ -22,8 +23,7 @@ export function getTestEntityRegistry() {
     entityRegistry.register(new DatasetEntity());
     entityRegistry.register(new UserEntity());
     entityRegistry.register(new TagEntity());
-    entityRegistry.register(new DataFlowEntity());
-    entityRegistry.register(new DataJobEntity());
```
Comment (gabe-lyons) [inline datahub-web-react/src/utils/test-utils/TestPageContainer.tsx:26]: whoops- looks like these lines should not have been deleted

File: datahub-web-react/src/app/entity/glossaryTerm/preview/__tests__/Preview.test.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import TestPageContainer from '../../../../../utils/test-utils/TestPageContainer';
+import { Preview } from '../Preview';
+
+describe('Preview', () => {
+    it('renders', () => {
+        const { getByText } = render(
+            <TestPageContainer>
+                <Preview urn="urn:li:tag:abc-sample-tag" name="name" definition="definition" owners={null} />
```
Comment (gabe-lyons) [inline datahub-web-react/src/app/entity/glossaryTerm/preview/__tests__/Preview.test.tsx:10]: why is the glossaryterm test accepting a tag urn?

File: datahub-graphql-core/src/main/resources/gms.graphql [hunk 3]
```diff
@@ -2,2253 +2,2288 @@
 scalar Long
 
 schema {
-    query: Query
-    mutation: Mutation
+  query: Query
+  mutation: Mutation
 }
```
Comment (gabe-lyons) [inline datahub-graphql-core/src/main/resources/gms.graphql:7]: This diff is very hard to read- could you revert the whitespace changes?

**Review by gabe-lyons — COMMENTED** [review]
looks good- left a few small comments but overall LGTM 👍

**Comment by saxo-lalrishav** [comment]
@gabe-lyons sorry for this PR, as this PR has too much conflict so i have created an another one and closing this one.
https://github.com/linkedin/datahub/pull/2538

File: datahub-web-react/src/app/entity/glossaryTerm/preview/__tests__/Preview.test.tsx (hunk 2 above)
Comment (saxo-lalrishav) [reply datahub-web-react/src/app/entity/glossaryTerm/preview/__tests__/Preview.test.tsx:10]: Fixed as part of https://github.com/linkedin/datahub/pull/2538

**Pull request closed without merging**